packages/slice-machine/src/domain/fields.ts

```diff
@@ -0,0 +1,202 @@
+import { IconName } from "@prismicio/editor-ui";
+import {
+  BooleanFieldType,
+  ColorFieldType,
+  DateFieldType,
+  EmbedFieldType,
+  FieldType,
+  GeoPointFieldType,
+  GroupFieldType,
+  ImageFieldType,
+  LinkFieldType,
+  NestableFieldTypes,
+  NumberFieldType,
+  RichTextFieldType,
+  SelectFieldType,
+  TextFieldType,
+  TimestampFieldType,
+  UIDFieldType,
+} from "@prismicio/types-internal/lib/customtypes";
+
+export interface Field {
+  description: string;
+  icon: IconName;
+  name: string;
+  type: FieldType;
+  variant?: FieldVariants;
```

Comment:
❓ does it make sense to export this base type?

Reply:
Nope you're right, mistake 👍

Reply:
At the end I updated it, the BaseField type no but Field type yes, used in AddFieldDropdown